ingest/indexer/service/indexer_streaming_service_test.go

```diff
@@ -411,6 +413,200 @@ func (s *IndexerServiceTestSuite) TestAddTokenLiquidity() {
 	}
 }
 
+func (s *IndexerServiceTestSuite) TestTrackCreatedPoolID() {
+	testCases := []struct {
+		name                     string    // Test case name
+		eventType                string    // Event type to be tested. Only poolmanagertypes.TypeEvtPoolCreated is valid type to be tracked
+		blockHeight              int64     // Block height to be used in the test
+		blockTime                time.Time // Block time to be used in the test
+		txnHash                  string    // Transaction hash to be used in the test
+		havePoolIDAttribute      bool      // Decide whether pool_id attribute should be appended in the event attributes during test data preparation
+		expectedError            bool      // Expected error flag
+		expectedPoolBeingTracked bool      // Expected whether the pool is being tracked
+	}{
+		{
+			name:                     "happy path",
+			eventType:                poolmanagertypes.TypeEvtPoolCreated,
+			blockHeight:              1000,
+			blockTime:                time.Now().UTC(),
+			txnHash:                  "txnhash",
+			havePoolIDAttribute:      true,
+			expectedPoolBeingTracked: true,
+		},
+		{
+			name:                     "Should not track with non pool_created event",
+			eventType:                gammtypes.TypeEvtTokenSwapped,
+			blockHeight:              1000,
+			blockTime:                time.Now().UTC(),
+			txnHash:                  "txnhash",
+			havePoolIDAttribute:      true,
+			expectedPoolBeingTracked: false,
+		},
+		{
+			name:                     "Should not track with no pool_id attribute",
+			eventType:                gammtypes.TypeEvtTokenSwapped,
+			blockHeight:              1000,
+			blockTime:                time.Now().UTC(),
+			txnHash:                  "txnhash",
+			havePoolIDAttribute:      false,
+			expectedPoolBeingTracked: false,
+		},
```

Comment:
For this test and all tests below, we should have `poolmanagertypes.TypeEvtPoolCreated` be set. Otherwise, how do we know that the reason we do not track stems from "no pool_id attribute"?

Reply:
GOOD CATCH! Sorry for the copy and paste errors.